frontend/src/loadEnvironment.js

```diff
@@ -25,11 +25,16 @@ const COLLAB_URL = SERVER_HOST + ':' + COLLAB_PORT;
 const HISTORY_PORT = process.env.REACT_APP_HISTORY_PORT || 9001;
 const HISTORY_URL = SERVER_HOST + ':' + HISTORY_PORT;
 
+// // Dependency Config: Execution   Service
```

Comment:
Looks like there's additional spacing between 'Execution' and 'Service'.

Reply:
Resolved